src/runtime/os_linux.go

```diff
@@ -139,3 +141,7 @@ func hardwareRand() (n uint64, ok bool) {
 //
 //export getrandom
 func libc_getrandom(buf unsafe.Pointer, buflen uintptr, flags uint32) uint32
+
+func fcntl(fd, cmd, arg int32) (ret int32, errno int32){
+    return 0, 0
+}
```

Comment:
This is a bad idea. It should not just ignore the call and return without an error.
Is this intended to call `syscall_fcntl` perhaps?

And then, any reason not to call the libc function instead (like with `syscall_Getpagesize` for example).